internal/pkg/skuba/deployments/ssh/files.go

```diff
@@ -29,16 +29,16 @@ func (t *Target) UploadFileContents(targetPath, contents string) error {
 	klog.V(1).Infof("uploading to remote file %q with contents", targetPath)
 	dir, _ := path.Split(targetPath)
 	encodedContents := base64.StdEncoding.EncodeToString([]byte(contents))
-	if _, _, err := t.silentSsh("mkdir", "-p", dir); err != nil {
+	if _, _, err := t.ssh("mkdir", "-p", dir); err != nil {
 		return err
 	}
-	_, _, err := t.silentSshWithStdin(encodedContents, "base64", "-d", "-w0", fmt.Sprintf("> %s", targetPath))
```

Comment:
I'm not so sure about this. We are doing this trick to upload/download any kind of content.

The intent here is that showing the contents of the files being uploaded/downloaded is not meaningful and might contain sensitive information (like base64'ed certificates and keys).

I think we want to keep the `silentSsh` logic when uploading and downloading files, as long as the error is clear if something fails.

Reply:
It is great to find it. I did not noticed that there should be the sensitive information.